File: FoodBack/src/main/java/cn/edu/fzu/sm2025/wwc/foodback/entity/FoodImage.java
package cn.edu.fzu.sm2025.wwc.foodback.entity;

import lombok.Data;
import lombok.EqualsAndHashCode;

import java.time.LocalDateTime;

@Data
public class FoodImage  {
    private Integer imageId;
    private Integer shopId;
    private String name;             // 图片展示的美食名称
    private String imagePath;
    private double price;            // 美食价格
    private String description;      // 美食描述


}

File: FoodPlatform/entry/src/main/ets/model/FoodImage.ts
// 基础图片接口
export interface Image {
  imageId: number;
  fileName: string;
  imagePath: string;
  uploadTime: string;
}

// 美食图片含美食基本信息接口
export interface FoodImage extends Image {
  shopId: number;
  name: string;
  price: number;
  description: string;
}